stdlib/public/core/Arrays.swift.gyb

```diff
@@ -1412,6 +1416,7 @@ extension Array {
   ///
   /// - Complexity: O(`self.count`) if the array is bridged,
   ///   otherwise O(1).
+  @discardableResult
   public mutating func popLast() -> Element? {
```

Comment:
LGTM. 

Sent from my moss-covered three-handled family gradunza

> On Apr 3, 2016, at 6:18 PM, Chris Lattner <notifications@github.com> wrote:
> 
> In stdlib/public/core/Arrays.swift.gyb:
> 
> > @@ -1412,6 +1416,7 @@ extension Array {
> >    ///
> >    /// - Complexity: O(`self.count`) if the array is bridged,
> >    ///   otherwise O(1).
> > +  @discardableResult
> >    public mutating func popLast() -> Element? {
> Good point, I'd appreciate @dabrahams thoughts on that.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
>